lib/functions/image/partitioning.sh

```diff
@@ -83,13 +83,9 @@ PRE_PREPARE_PARTITIONS
 	local next=1
 	# Check if we need UEFI partition
 	if [[ $UEFISIZE -gt 0 ]]; then
-		if [[ "${IMAGE_PARTITION_TABLE}" == "gpt" ]]; then
-			local uefipart=15
```

Comment:
What was the purpose of this partition number?

Reply:
> What was the purpose of this partition number?

The correct order of partition creation does not depend on the type of MBR. Section numbers should start with 1, no "last numbers" (14\15) should be used. If EFI is used, the first partition should be for EFI, the root partition goes by the next number, in the same way as it is implemented for a separate boot\root.

Reply:
I understand - I wonder if you perhaps know the origins of this magical number? Why 15 and not ... 17 ? :)

Reply:
This is not a question for me (I did not create these numbers)  :) 
That is why I have made all the changes (if possible) in a separate file.